### PR TITLE
docs/variants/msi_z690/recovery.md: add documented recovery methods to intro

### PR DESCRIPTION
**Pull request by nedpfeiffer**
Adding a concise overview of the two recovery methods helps orient the reader to the contents of the recovery documentation.

Signed-off-by: nedpfeiffer <116780681+nedpfeiffer@users.noreply.github.com>

### CODE REVIEW

File: docs/variants/msi_z690/recovery.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ hardware configurations, the Dasharo firmware may not boot correctly (i.e.
 we will have "bricked" the platform). In such a case, the recovery procedure can
 reinstall the original firmware from the board manufacturer.
 
+There are two documented recovery methods: using a CH341A programming kit or an
+RTE external programmer.
```
Comment (pietrushnic) [inline docs/variants/msi_z690/recovery.md:11]:
```suggestion
RTE.
```

Both CH341A and RTE are external SPI programmers

Maybe linking to those sections would be useful if this document will grow in future

Comment (nedpfeiffer) [reply docs/variants/msi_z690/recovery.md]: I think that's a good idea, I'll include it in my next PR.

**Review by miczyg1 — APPROVED** [review]

**Comment by miczyg1** [comment]
@nedpfeiffer your commit in this PR is unsigned, please fix it to unlock merging.

**Comment by nedpfeiffer** [comment]
Resubmitting this PR because my GPG key wasn't set up. Third time's the charm!

**Pull request closed without merging**